homeassistant/components/growatt_server/sensor.py

```diff
@@ -454,13 +454,13 @@
     ),
     "mix_wattage_pv_1": (
         "PV1 Wattage",
-        POWER_WATT,
+        POWER_KILO_WATT,
```

Comment:
We should mention this change as a breaking change so it's highlighted in the release notes.

Reply:
Done